core/src/impl/KokkosExp_Host_IterateTile.hpp

```diff
@@ -2324,15 +2321,10 @@ struct HostIterateTile<RP, Functor, Tag, ValueType,
                                         // 'array-ness' [], only
                                         // underlying type remains
 
-  inline HostIterateTile(
-      RP const& rp, Functor const& func,
-      value_type* v)  // v should be an array; treat as pointer for
-                      // compatibility since size is not known nor needed here
-      : m_rp(rp)      // Cuda 7.0 does not like braces...
+  inline HostIterateTile(RP const& rp, Functor const& func)
+      : m_rp(rp)  // Cuda 7.0 does not like braces...
```

Comment:
It's a silly thing, but we really don't care about Cuda 7 anymore

Reply:
Yeah but I didn't want to do unrelated changed.